context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to create art or 
+images on the computer using code. 
 
 ## What is easy to do in this language? Why is it easy?
 
+Of course, it is very easy to create images. More specifically, it is easy to 
+create images with repeating components.  By defining rules, it is very easy to 
+create individual shapes which can be called and used in other parts of the image. 
+Furthermore, since users can call the rules in other parts of the code,
+recursion is relatively easy and shapes can be reused in different variations 
+again easily. Though saying it is easy to create images is very broad, what 
+comes to mind is how much cleaner it is to read in comparison to SVG, which is 
+widely used for creating images on the web. 
 
 ## What is hard to do in this language? Why is it hard?
 
+It is difficult to get the exact shape and color you want since the shape 
+adjustments are done through numerical calculations, especially on the first try.
+Getting pretty much any of these variations required some trial and error, which
+should not really be the case for anything mathematical or code-based.  Though
+code should make it very clear what the outcome is, this principle does not 
+translate nicely to visual things such as image adjustments. If there was a way 
+to adjust the code using a method that interacts with the image more directly 
+(for example, a color slider), it would be much easier and more intuitive to 
+learn how to actually write the proper code for certain shapes. 
```

Comment:
I shared a similar feeling about the lack of easy precision when using ContextFree. Ultimately, if I wanted to make something based off of another image or an end-goal, I'd likely want to use something like Adobe Illustrator that offers a GUI experience&mdash;something I think lends itself much better to visual design than writing code.